mlflow/tracking/fluent.py

```diff
@@ -830,6 +830,28 @@ def set_experiment_tag(key: str, value: Any) -> None:
     MlflowClient().set_experiment_tag(experiment_id, key, value)
 
 
+def delete_experiment_tag(key: str) -> None:
+    """
+    Delete a tag from the current experiment.
+
+    Args:
+        key: Tag name. This string may only contain alphanumerics, underscores (_), dashes (-),
+            periods (.), spaces ( ), and slashes (/). All backend stores will support keys up to
+            length 250, but some may support larger keys.
+
+    .. code-block:: python
+        :test:
+        :caption: Example
+
+        import mlflow
+
+        with mlflow.start_run():
+            mlflow.delete_experiment_tag("release.version")
```

Comment:
could we update the example to also set a tag? generally i think it's good if the documentation examples are self-contained. i think the `start_run()` is also unnecessary here to illustrate the behavior
```suggestion
        import mlflow

        mlflow.set_experiment("test-delete-tag")

        mlflow.set_experiment_tag("release.version", "1.0")
        mlflow.delete_experiment_tag("release.version")
```